src/main/java/com/noroff/mefit/backup/BackUp.java

```diff
@@ -0,0 +1,83 @@
+package com.noroff.mefit.backup;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Calendar;
+import java.util.Map;
+
+public class BackUp {
+    public static void DumpBackup() {
+        Process p;
+        ProcessBuilder pb;
+        pb = new ProcessBuilder(
+                "C:\\Program Files\\PostgreSQL\\14\\bin\\pg_dump.exe",
+                "--host=localhost",
+                "--port=5432",
+                "--username=postgres",
+                "--no-password",
+                "--format=custom",
+                "--blobs",
+                "--verbose", "--file", "C:\\Users\\OmaAli\\{data_source}-{timestamp}-dump.sql", "postgres");
+        try {
+            final Map<String, String> env = pb.environment();
+            env.put("PGPASSWORD", "admin");
+            p = pb.start();
+            final BufferedReader r = new BufferedReader(
+                    new InputStreamReader(p.getErrorStream()));
+            String line = r.readLine();
+            while (line != null) {
+                System.err.println(line);
+                line = r.readLine();
+            }
+            r.close();
+            p.waitFor();
+            System.out.println(p.exitValue());
+
+        } catch (IOException | InterruptedException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public static void RestoreBackup() throws IOException {
+        Runtime r = Runtime.getRuntime();
+        Process p;
+        ProcessBuilder pb;
+        pb = new ProcessBuilder(
+                "C:/Program Files/PostgreSQL/14/bin/pg_restore.exe",
+                "--host=localhost",
+                "--port=5432",
+                "--username=postgres",
+                "--no-password",
+                "--dbname=postgres",
+                "--section=pre-data",
+                "--section=data",
+                "--section=post-data",
+                "--verbose", "C:\\Users\\OmaAli\\{data_source}-{timestamp}-dump.sql");
```

Comment:
Same questions as above

Reply:
Yes, it does, its just that i wanted the backup to work locally first